packages/blade/src/components/Input/BaseInput/BaseInput.tsx

```diff
@@ -282,6 +285,20 @@ type BaseInputCommonProps = FormInputLabelProps &
      * State setter for active tag index
      */
     setActiveTagIndex?: (activeTagIndex: number) => void;
+    /**
+     * Sets the size of the input field
+     * @default medium
+     */
+    size?: 'medium' | 'large';
+    /**
+     * Link button to be rendered at the end of the input field.
+     * **Note:** `size` of the Link will be set to the same size as the input field, `isDisabled` will follow Input's `isDisabled`, & `variant` will be set to `button`.
+     * Example:
+     * ```tsx
+     * trailingActionButton={<Link onClick={handleClick}>Apply</Link>}
+     * ```
+     */
+    trailingActionButton?: React.ReactElement<LinkProps>;
```

Comment:
I remember discussing this at office, did we decide on trailingActionButton? 🤔

Reply:
Yeah. Earlier it was `trailingLinkButton`. After our discussion made it `trailingActionButton`

Reply:
Ah right makes sense.

Reply:
What if we just do `trailingButton` or `trailingAction`? Button implies Action no?

Reply:
`trailingButton` makes more sense. Action is still very open ended. Changed it to `trailingButton`